### PR TITLE
Add interfaces for V-probe data

### PR DESCRIPTION
**Pull request by felixp8**
Add custom interface for `.dat` file V-probe continuous data and metadata handling

### CODE REVIEW

**Comment by felixp8** [comment]
example file here: https://drive.google.com/file/d/1vKsreW2WX7tpZ4GsrL6uhbBvIj2QozLb/view?usp=sharing

**Comment by CodyCBakerPhD** [comment]
Looks like the conversion factors for the electrical series are the default, `1e-6`: probably not correct, and need to set that to the proper value before we can merge this

**Comment by felixp8** [comment]
The `.dat` files are actually in microvolts (converted from the raw OpenEphys data's 0.195 uV resolution in [this line](https://github.com/nwatters01/catalystneuro/blob/main/phys_preprocessing/open_ephys_utils/load_open_ephys_data.m#L428) but kept in int16) so I think that value is correct, unfortunately...

**Comment by CodyCBakerPhD** [comment]
Thanks for clarifying - we should check with them that they are aware of the data loss, and even compute/evaluate that ourselves

But I guess that means this PR could technically be good to go once the conflicts are resolved

**Comment by CodyCBakerPhD** [comment]
ping @felixp8

**Comment by felixp8** [comment]
no need to merge yet, I need to add some fixes for overlapping unit names real quick

**Comment by CodyCBakerPhD** [comment]
> no need to merge yet, I need to add some fixes for overlapping unit names real quick

OK - returned to draft - reopen and request me when ready

**Comment by felixp8** [comment]
lumped a couple changes into this one:
* adding the v-probe data interfaces
* splitting out raw and processed files in `session_to_nwb`
* update/reorganize `README.md` to have more detailed instructions specific to Watters data conversion

here are the two stub files: https://drive.google.com/drive/folders/1ZVu83AFRW0ZeleHKWB-Tg0utlXg2pkhE?usp=drive_link

File: src/jazayeri_lab_to_nwb/watters/watters_convert_session.py [hunk 1]
```diff
@@ -19,53 +19,102 @@ def session_to_nwb(data_dir_path: Union[str, Path], output_dir_path: Union[str,
...
 
     # Add LFP
     lfp_files = list(glob.glob(str(data_dir_path / "raw_data" / "spikeglx" / "*" / "*" / "*.lf.bin")))
     assert len(lfp_files) > 0, f"No .lf.bin files found in {data_dir_path}"
     assert len(lfp_files) == 1, f"Multiple .lf.bin files found in {data_dir_path}"
-    source_data.update(dict(LFP=dict(file_path=str(lfp_files[0]), es_key="ElectricalSeriesLF")))
-    conversion_options.update(dict(LFP=dict(write_as="lfp", stub_test=stub_test)))
+    raw_source_data.update(dict(LFP=dict(file_path=str(lfp_files[0]))))
+    processed_source_data.update(dict(LFP=dict(file_path=str(lfp_files[0]))))
+    raw_conversion_options.update(dict(LFP=dict(write_as="lfp", stub_test=stub_test)))
```
Comment (CodyCBakerPhD) [inline src/jazayeri_lab_to_nwb/watters/watters_convert_session.py]: So, this is actually the one exception where we do not write it as LFP - for SpikeGLX, the LF band is acquired simultaneously  with the AP so it counts as raw acquisition

Most of the time, other systems process this offline

```suggestion
    raw_conversion_options.update(dict(LF=dict(stub_test=stub_test)))
```

File: src/jazayeri_lab_to_nwb/watters/wattersnwbconverter.py [hunk 2]
```diff
@@ -26,6 +30,10 @@ class WattersNWBConverter(NWBConverter):
     """Primary conversion class for my extracellular electrophysiology dataset."""
 
     data_interface_classes = dict(
+        RecordingVP0=WattersDatRecordingInterface,
+        SortingVP0=KiloSortSortingInterface,
+        RecordingVP1=WattersDatRecordingInterface,
+        SortingVP1=KiloSortSortingInterface,
         RecordingNP=SpikeGLXRecordingInterface,
         LFP=SpikeGLXRecordingInterface,
```
Comment (CodyCBakerPhD) [inline src/jazayeri_lab_to_nwb/watters/wattersnwbconverter.py]: Sorry for not catching this earlier; I'd rename this 'LF'

Will need to update all references elsewhere to this key

```suggestion
        LF=SpikeGLXRecordingInterface,
```

**Comment by CodyCBakerPhD** [comment]
Files look good except for that LFP classification from SpikeGLX - should be an easy fix then this is good to go

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
Awesome, thanks!

**Pull request merged**